delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/Resources.scala

```diff
@@ -293,72 +316,104 @@ object Resources {
         case _ => IO.raiseError(ResourceAlreadyExists(c.id, c.project))
       }
 
-    def update(c: UpdateResource) =
+    def validateStateResourceExists(c: ModifyCommand) = {
```

Comment:
`resourceExists` ?

Reply:
stateWhereResourceExists? As it is returning a state where that is true, or a reason why it isn't

Reply:
Ok for me :)